### PR TITLE
'case mismatch $VERSION\$version in ci-test; added ! to hashbang jill.sh'

### PR DESCRIPTION
**Pull request by sdotd**

### CODE REVIEW

File: ci-test.sh [hunk 1]
```diff
@@ -20,11 +20,11 @@ msg "Testing julia-x-y is installed"
...
 
 msg "Installing specific Julia version"
-echo "yy" | bash jill.sh --version $VERSION
-julia-$VERSION -v
+echo "yy" | bash jill.sh --version $version
+julia-$version -v
 
 msg "Testing if the version is correct"
-[[ $(julia-$VERSION -v) == "julia version $VERSION" ]]
+[[ $(julia-$version -v) == "julia version $version" ]]
```
Comment (abelsiqueira) [inline ci-test.sh:27]: `VERSION` is the variable set inside GitHub Actions, you can leave it that way.

File: jill.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#/usr/bin/env bash
+#!/usr/bin/env bash
```
Comment (abelsiqueira) [inline jill.sh:1]: I was surprised that it worked without it. TIL it defaults to `/bin/sh`.

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Thanks. Could you remove the other modification of VERSION, please.

File: ci-test.sh (hunk 1 above)
Comment (sdotd) [reply ci-test.sh:27]: oh ok, I was using it to run the install script and it kept creating bad links. sorry about that.

Comment (sdotd) [reply ci-test.sh]: removed my ci-test changes.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**